app/controllers/api/search_filters_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  class SearchFiltersController < BaseController
+    def delete_resource(type, id, data = {})
+      id ||= data['description']
```

Comment:
we could add identifying_attrs: of description in the api.yml and this would be done for us by the manager.